### PR TITLE
WIP: Onboard ACM scenario

### PR DESCRIPTION
**Pull request by ascerra**

### CODE REVIEW

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-ascerra-clc-ui-e2e-release-2.7-acm-ocp4.12-lp-interop-acm-interop-aws

File: ci-operator/step-registry/acm/mch/acm-mch-commands.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# can't wait before the resource exists. Need to sleep a bit before start watching
+sleep 60
+
+RETRIES=30
+for i in $(seq "${RETRIES}"); do
+  if [[ $(oc get mch -n ocm -o=jsonpath='{.items[0].status.phase}') == "Running" ]]; then
+    echo "MCH is Running"
+    break
+  else
```
Comment (calebevans) [inline ci-operator/step-registry/acm/mch/acm-mch-commands.sh:30]: You may want to add a check in this else statement to see if you are on the last retry, if you are exit the script with an error.

Here is what I did with MTR:

```bash
RETRIES=30
for try in $(seq "$RETRIES"); do
    DEPLOYMENT_EXISTS=$(oc get windup -n mtr -o=jsonpath='{.items[0].status.conditions[0].status}')
    READY=$(oc get windup -n mtr -o=jsonpath='{.items[0].status.conditions[1].status}')

    if [ DEPLOYMENT_EXISTS == "True" ] && [ READY == "True"]; then
        echo "Windup deployment exists and is ready."
        break
    else
        if [ $try == $RETRIES ]; then
            echo "Error deploying Windup, exiting now"
            exit 1
        fi
        echo "Try ${try}/${RETRIES}: Windup deployment is not ready. Checking again in 30 seconds"
        sleep 30
    fi
done
```

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

File: ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+export HUB_OCP_API_URL
+HUB_OCP_PASSWORD=$(cat $SHARED_DIR/kubeadmin-password)
+export HUB_OCP_PASSWORD
+
+# run the test execution script
+./fetch_clusters_commands.sh
+
+if [ -s /tmp/ci/managed.cluster.name ] && ! grep -q "null" /tmp/ci/managed.cluster.name; then
+    echo "managed.cluster.name file was found saving as artifact."
+    cp -r /tmp/ci/managed.cluster.name $ARTIFACT_DIR/
```
Comment (ascerra) [inline ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh:19]: cp -r /tmp/ci/managed.cluster.name $SHARED_DIR/
cp -r /tmp/ci/managed.cluster.url $SHARED_DIR/
cp -r /tmp/ci/managed.cluster.password $SHARED_DIR/
cp -r /tmp/ci/managed.cluster.etc $SHARED_DIR/

use $SHARED_DIR/managaedCluster.json

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Comment by ascerra** [comment]
/pj-rehearse periodic-ci-stolostron-acmqe-autotest-main-acm-ocp4.12-lp-interop-acm-interop-aws

**Pull request closed without merging**